src/dilovar/Maps/MapFrequencyOfCharacters.java

```diff
@@ -46,4 +47,36 @@ public static Map<Character,Integer> frequencyOfCharacters(String str){
 
         return noDuplicates;
     }
+    public static void frequencyOfCharacters2(String str){
```

Comment:
Not sure about the purpose of this method

You could utilize Set for containing unique keys and Collections.frequency(Collection c, Object o) to have the frequency of each letter from a list containing all letters, or use set with simple loop to count each letter and store.